src/modal/modal-window.ts

```diff
@@ -97,7 +97,7 @@ export class NgbModalWindow implements OnInit,
     } else {
       elementToFocus = body;
     }
-    elementToFocus.focus();
+    setTimeout(() => elementToFocus.focus());
```

Comment:
do we want to run it in the Angular zone and trigger additional CD after destroy?

Reply:
No a side note it wont change anything for this particular PR, because the whole focus trap runs inside Angular zone oO. I'll fix that in a separate PR.